tests/components/maxcube/conftest.py

```diff
@@ -0,0 +1,132 @@
+"""Tests for EQ3 Max! component."""
+from unittest.mock import create_autospec, patch
+
+from maxcube.device import MAX_DEVICE_MODE_AUTOMATIC, MAX_DEVICE_MODE_MANUAL
+from maxcube.room import MaxRoom
+from maxcube.thermostat import MaxThermostat
+from maxcube.wallthermostat import MaxWallThermostat
+from maxcube.windowshutter import MaxWindowShutter
+import pytest
+
+from homeassistant.components.homeassistant import (
+    DOMAIN as HA_DOMAIN,
+    SERVICE_UPDATE_ENTITY,
+)
+from homeassistant.components.maxcube import DOMAIN
+from homeassistant.const import ATTR_ENTITY_ID
+
+from tests.common import get_test_home_assistant, setup_component
+
+
+@pytest.fixture
+def room():
+    """Create a test MAX! room."""
+    r = MaxRoom()
+    r.id = 1
+    r.name = "TestRoom"
+    return r
+
+
+@pytest.fixture
+def thermostat():
+    """Create test MAX! thermostat."""
+    t = create_autospec(MaxThermostat)
+    t.name = "TestThermostat"
+    t.serial = "AABBCCDD01"
+    t.rf_address = "abc1"
+    t.room_id = 1
+    t.is_thermostat.return_value = True
+    t.is_wallthermostat.return_value = False
+    t.is_windowshutter.return_value = False
+    t.mode = MAX_DEVICE_MODE_AUTOMATIC
+    t.comfort_temperature = 19.0
+    t.eco_temperature = 14.0
+    t.target_temperature = 20.5
+    t.actual_temperature = 19.0
+    t.max_temperature = None
+    t.min_temperature = None
+    t.valve_position = 25  # 25%
+    return t
+
+
+@pytest.fixture
+def wallthermostat():
+    """Create test MAX! wall thermostat."""
+    t = create_autospec(MaxWallThermostat)
+    t.name = "TestWallThermostat"
+    t.serial = "AABBCCDD02"
+    t.rf_address = "abc2"
+    t.room_id = 1
+    t.is_thermostat.return_value = False
+    t.is_wallthermostat.return_value = True
+    t.is_windowshutter.return_value = False
+    t.mode = MAX_DEVICE_MODE_MANUAL
+    t.comfort_temperature = 19.0
+    t.eco_temperature = 14.0
+    t.target_temperature = 4.5
+    t.actual_temperature = 19.0
+    t.max_temperature = 29.0
+    t.min_temperature = 4.5
+    return t
+
+
+@pytest.fixture
+def windowshutter():
+    """Create test MAX! window shutter."""
+    shutter = create_autospec(MaxWindowShutter)
+    shutter.name = "TestShutter"
+    shutter.serial = "AABBCCDD03"
+    shutter.rf_address = "abc3"
+    shutter.room_id = 1
+    shutter.is_open = True
+    shutter.is_thermostat.return_value = False
+    shutter.is_wallthermostat.return_value = False
+    shutter.is_windowshutter.return_value = True
+    return shutter
+
+
+@pytest.fixture
+def hass_config():
+    """Return test HASS configuration."""
+    return {
+        DOMAIN: {
+            "gateways": [
+                {
+                    "host": "1.2.3.4",
+                }
+            ]
+        }
+    }
+
+
+@pytest.fixture
+def hass():
+    """Mock hass core."""
+    hass = get_test_home_assistant()
+    yield hass
+    hass.stop()
+
+
```

Comment:
We already have a default `hass` fixture.
```suggestion
```

Reply:
Removed